keel-core/src/test/kotlin/com/netflix/spinnaker/keel/actuation/EnvironmentPromotionCheckerTests.kt

```diff
@@ -174,7 +174,7 @@ internal class NewEnvironmentPromotionCheckerTests : JUnit5Minutests {
 
           test("an event is not sent") {
             verify(exactly = 0) {
-              publisher.publishEvent(any<ArtifactVersionApproved>())
+              publisher.publishEvent(ofType<ArtifactVersionApproved>())
```

Comment:
Turns out that's not how you do it

Reply:
oh interesting

Reply:
Yeah, I fixed a bunch more in https://github.com/spinnaker/keel/pull/1631

Reply:
`any()` is fine in some situations, but if you want to strictly match the type of the parameter that actually gets passed, that's not what it does.